### PR TITLE
Changed only some comments, in order to leave the later logic as it is

### PR DESCRIPTION
**Pull request by joaquinfilipic-dynatrace**

### CODE REVIEW

File: k8s/helm-chart/dynatrace-gcp-function/values.yaml [hunk 1]
```diff
@@ -13,37 +13,41 @@
...
-# if set to logs - deploy only one container to send GCP logs to Dynatrace
-# if set to all - deploy two containers to send GCP metrics and logs to Dynatrace
+# if set to 'metrics' - deploy only one container to send GCP metrics to Dynatrace
+# if set to 'logs' - deploy only one container to send GCP logs to Dynatrace
+# if set to 'all' - deploy two containers to send GCP metrics and logs to Dynatrace
 deploymentType: "all"
-# Dynatrace API token with permissions:
-# Deployment type 'all' or'logs'
+# Dynatrace API token with permissions, according to the deployment type
+# - for deployment type 'logs':
```
Comment (wojtasskorcz) [inline k8s/helm-chart/dynatrace-gcp-function/values.yaml:25]: The `all` option should still be mentioned here and below

File: k8s/helm-chart/dynatrace-gcp-function/values.yaml [hunk 2]
```diff
@@ -13,37 +13,41 @@
...
 dynatraceUrl: ""
 
+
 # LOGS VALUES - REQUIRED for 'logs' and 'all' deployment
 
 # subscription id of log sink pubsub subscription
 logsSubscriptionId: ""
 #dynatraceLogIngestUrl endpoint used to ingest logs to Dynatrace:
-#  - for direct ingest through the Cluster API: https://<your_environment_ID>.live.dynatrace.com
+#  - for direct ingest through the Cluster API: https://<your_environment_id>.live.dynatrace.com
```
Comment (wojtasskorcz) [inline k8s/helm-chart/dynatrace-gcp-function/values.yaml:43]: We have `<environment_id>` above. I think we should stick to one naming convention

File: k8s/helm-chart/dynatrace-gcp-function/values.yaml [hunk 3]
```diff
@@ -13,37 +13,41 @@
...
-# Deployment type 'all' or 'metrics'
+# - for deployment type 'metrics':
 # API v2: 'metrics.ingest', 'extensions.read', 'extensions.write', 'extensionConfigurations.read', 'extensionConfigurations.write', 'extensionEnvironment.read', 'extensionEnvironment.write'
 # API v1: 'ReadConfig', 'WriteConfig'
+# - for deployment type 'all':
+# All the above permissions
 dynatraceAccessKey: ""
-# Dynatrace environment endpoint, for example: SaaS: https://environment-id.live.dynatrace.com, Managed: https://cluster-id.managed.internal.dynatrace.com/e/environment-id
-# Needed for deployment type 'metrics'. For deployment 'all' and 'logs' in case when activeGate.useExisting is false
+# Dynatrace environment endpoint, for example: SaaS: https://<environment_id>.live.dynatrace.com, Managed: https://<cluster_id>.managed.internal.dynatrace.com/e/<environment_id>
```
Comment (dtwwolinski) [inline k8s/helm-chart/dynatrace-gcp-function/values.yaml:33]: Cosider bellow:
```suggestion
# Dynatrace environment endpoint, for example: 
- SaaS: https://<environment_id>.live.dynatrace.com, 
- Managed: https://<cluster_id>.managed.internal.dynatrace.com/e/<environment_id>
```

**Review by dtwwolinski — DISMISSED** [review]

File: k8s/helm-chart/dynatrace-gcp-function/values.yaml (hunk 3 above)
Comment (joaquinfilipic-dynatrace) [reply k8s/helm-chart/dynatrace-gcp-function/values.yaml]: Done.

**Review by wojtasskorcz — APPROVED** [review]

**Review by dtwwolinski — APPROVED** [review]

File: k8s/helm-chart/dynatrace-gcp-function/values.yaml (hunk 2 above)
Comment (wojtasskorcz) [reply k8s/helm-chart/dynatrace-gcp-function/values.yaml]: Yes, I've noticed :)

**Pull request merged**